modules/preparing-an-inital-cluster-deployment-for-mce.adoc

```diff
@@ -0,0 +1,401 @@
+// Module included in the following assemblies:
+//
+// * installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-an-inital-cluster-deployment-for-mce_{context}"]
+
+= Preparing an initial cluster deployment for multicluster engine for Kubernetes(MCE) Operator
+
+The following procedure explains how you can:
+
+* Deploy an initial cluster with the Agent-based Installer and use that cluster as the hub cluster.
+* Deploy the MCE Operator.
+* Install the Red Hat Advanced Cluster Management (RHACM) which deploys the assisted service.
+
+.Procedure
+
+. Create a `<manifests_directory>` folder to place a valid `install-config.yaml` and `agent-config.yaml` file. This directory is useed to store all the assets.
+
+. Create a `ImageSetConfiguration.yaml` file with the following settings:
```

Comment:
This part of the procedure is related only when performing the installation for a disconnected environment. It's a series of addtional steps, and this fact should be highlighted somehow (and I'd suggest to keep it separated from the standard procedure, which assumes a connected environment, and where effectively the required steps are described)